src/main/java/com/nha/abdm/wrapper/common/RequestManager.java

```diff
@@ -11,16 +11,24 @@
 @Component
 public class RequestManager {
 
-  @Autowired SessionManager sessionManager;
+  private WebClient webClient;
+
+  public RequestManager(
```

Comment:
Is `@Autowired` annotation not need on the constructor method. Because I see that we need to inject the `SessionManager`?

Reply:
made the changes, thanks. Its getting injected this way as well.

Reply:
Oh Ok